pyrebase/pyrebase.py

```diff
@@ -1,27 +1,26 @@
-import requests
-from requests import Session
-from requests.exceptions import HTTPError
-
-try:
-    from urllib.parse import urlencode, quote
-except:
-    from urllib import urlencode, quote
+from collections import OrderedDict
 import json
 import math
 from random import uniform
-import time
-from collections import OrderedDict
-from sseclient import SSEClient
-import threading
 import socket
-from oauth2client.service_account import ServiceAccountCredentials
-from gcloud import storage
+import threading
+import time
+
+from google.auth import jwt
+from google.auth.transport.requests import AuthorizedSession
+from google.cloud import storage
+from google.oauth2 import service_account
+import requests
+from requests.exceptions import HTTPError
 from requests.packages.urllib3.contrib.appengine import is_appengine_sandbox
 from requests_toolbelt.adapters import appengine
 
-import python_jwt as jwt
-from Crypto.PublicKey import RSA
-import datetime
+from sseclient import SSEClient
+
+try:
+    from urllib.parse import urlencode, quote
+except ImportError:
+    from urllib import urlencode, quote
 
 
 def initialize_app(config):
```

Comment:
for example:

```python
def initialize_app(config, session=None):
    if session is not None:
        self.requests = session
    else:
        ...
````

In this case we can use our own session.

Reply:
@lepture That's a good idea. However, I think it's a bit outside the scope of this PR. It would change the external API, and I was trying not to do that as part of this change. I'm intending for `google-auth` to be a drop-in replacement for `oauth2client`.